### DISCUSSION TITLE
Auto conversion of hex codes into RGB

### DISCUSSION THREAD

**Discussion by malpeli** [vasqued2/ha-teamtracker#31] [Ideas]
Love your work. I would love it even more in future if the team/opponents colours could be automatically converted into RGB values for LED lights to react to. Would be super cool to be able to have my living room flash with the colours of the team that just scored, or perhaps have an addressable LED strip behind the tv that glows in the colours of both teams according to their likelihood of victory. Just a thought

**Comment by vasqued2** [comment]
Can't you just extract the RGB values or am I missing something?

```
{% states.sensor.tt_teamtracker.attributes.team_colors[0][1:3] | int(base=16) %}
{% states.sensor.tt_teamtracker.attributes.team_colors[0][3:5] | int(base=16) %}
{% states.sensor.tt_teamtracker.attributes.team_colors[0][5:8] | int(base=16) %}
```
